### PR TITLE
New package: execline-2.1.2.2

### PR DESCRIPTION
**Pull request by bougyman**

### CODE REVIEW

File: srcpkgs/execline/template [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+maintainer="bougyman <bougyman@voidlinux.eu>"
+license="ISC"
+homepage="http://skarnet.org/software/execline/"
+distfiles="http://skarnet.org/software/${pkgname}/${pkgname}-${version}.tar.gz"
+checksum=aea08c2c1b986c91f300d40737dd43067b91705d2c729859344ec65f989aec06
+
+post_install() {
+	vlicense COPYING
+	vdoc README
+	vcopy doc/* usr/share/doc/${pkgname}
```
Comment (beefcurtains) [inline srcpkgs/execline/template]: ``"doc/*"``, no?

File: srcpkgs/execline/template [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+}
+
+execline-devel_package() {
+	short_desc+=" - development files"
+	depends="${sourcepkg}>=${version}_${revision}"
+	pkg_install() {
+		vmove usr/include
+		vmove usr/lib
```
Comment (beefcurtains) [inline srcpkgs/execline/template]: Pardon me, but sure it's not ``"usr/lib/*.so"``?

Comment (bougyman) [inline srcpkgs/execline/template]: the only thing in usr/lib/ is usr/lib/execline/libexecline.a. Should I specify the full file path, do you think?

File: srcpkgs/execline/template (hunk 1 above)
Comment (bougyman) [inline srcpkgs/execline/template]: It worked without quotes, but I will add.

File: srcpkgs/execline/template (hunk 2 above)
Comment (beefcurtains) [inline srcpkgs/execline/template]: In that case, might as well pass ``--disable-static`` to ``configure_args``.

File: srcpkgs/execline/template [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Template file for 'execline'
+pkgname=execline
+version=2.1.2.2
+revision=1
+build_style=gnu-configure
+makedepends="skalibs-devel"
```
Comment (xtraeme) [inline srcpkgs/execline/template]: This pkg needs to set `configure_args="--libdir=/usr/lib"` and mv the static libs to -devel.

Comment (bougyman) [inline srcpkgs/execline/template]: Done, rebased, and squashed.

**Comment by xtraeme** [comment]
Looks ok now :+1:

**Pull request merged**